sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md

```diff
@@ -1,17 +1,11 @@
 # Release History
 
-## 7.6.0-beta.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 7.5.1 (2021-12-08)
```

Comment:
I think the azure-core version was also updated which should be included in the changelog

Reply:
Done